src/main/java/com/google/sps/data/NSWGovAQDataPoint.java

```diff
@@ -7,10 +7,10 @@
  */
 public class NSWGovAQDataPoint {
   @SerializedName(value = "Site_Id")
-  private final int siteId;
+  public int siteId;
   @SerializedName(value = "Value")
```

Comment:
nit: should we also give it a consistent name, `Value` vs `aqi`, @purple-waves wdyt?

Reply:
The `@SerializedName` needs to reflect the JSON string that this arrives as.

Calling the data member `aqi` is much more descriptive; especially if we later extend this to include more 'vaules' than just AQI (for example, passing through PM2.5 and PM10 values as well).

Reply:
Yep, that's what I was about to say

Reply:
@Techget it is a "Value" because this is being populated by the post request that fetches the data from the Government API, and the variable name for the AQI value there is called "Value"

Reply:
okay, thanks for the info